### ISSUE TITLE
[Question] 其实私聊也能在普通服务器下使用

### ISSUE THREAD

**Issue by Night-GUA** [g0aty/SickoMenu#143]
那么，会把私聊改成SafeMode可用吗

**Comment by g0aty** [comment]
Yes, whisper will be changed to work with safe mode, it accidentally got removed a while ago.

Translation (according to Google Translate):
是的，whisper 将在安全模式下运行，它不久前被意外删除了
